motion_guide.html

```diff
@@ -3395,8 +3379,7 @@
 
 <p></p>
 The video4linux driver is written by the same author that first created Motion. You can 
-find the source and a brief description at the 
-<a href="http://www.lavrsen.dk/twiki/bin/view/Motion/VideoFourLinuxLoopbackDevice" class="twikiLink">video4linux loopback device web page</a>. 
+find the source within apt packages.  
```

Comment:
I can't actually find the loopback device anywhere other than the old wiki; it used to be packaged in the various linux distress but they all seem to have removed the packages.

I think the two options here are:

1) Remove this section entirely
2) Leave it pointing at the old wiki

I'm probably slightly veering towards 1, unless we find there is a core of users still using it.